src/landpatterns/numbering.stanza

```diff
@@ -41,6 +41,17 @@ Convert the row and column indices into a pad number or reference
 <DOC>
 public defmulti to-pad-id (x:Numbering, row:Int, column:Int) -> Int|Ref
 
+doc: \<DOC>
+Null Numbering Object
+
+This is a default object for creating packages that don't require
+any pad numbering. These landpatterns may not contain any pads to
```

Comment:
Is this `may not` or `should not`?

Reply:
This doesn't prevent composition of landpatterns or manually defining, for example, a mechanical hole pad outside of these constructs.

Reply:
Cool, then maybe we change the language to something like? 
"These landpatterns do not need to contain any pads to number."